### PR TITLE
346 : Add photo URL field in product Catalogue domain.

### PR DESCRIPTION
**Pull request by rehammuzzamil**
https://github.com/OpenSRP/opensrp-server-core/issues/346

### CODE REVIEW

File: src/test/java/org/opensrp/service/ProductCatalogueServiceTest.java [hunk 1]
```diff
@@ -60,6 +60,7 @@ public void testUpdate() {
 		productCatalogueService.update(productCatalogue);
 		ProductCatalogue updatedProductCatalogue = productCatalogueService.getProductCatalogue(1l);
 		assertEquals("Updated Product Name", updatedProductCatalogue.getProductName());
+		assertEquals("", updatedProductCatalogue.getPhotoURL());
```
Comment (githengi) [inline src/test/java/org/opensrp/service/ProductCatalogueServiceTest.java:63]: Can we update the URL and test here

Comment (rehammuzzamil) [reply src/test/java/org/opensrp/service/ProductCatalogueServiceTest.java:63]: URL can not be updated, as URL points to the end-point /multimedia/media/{entity-id} to download the image file where entity-id is equaled to the unique id of the product catalogue.

Comment (githengi) [reply src/test/java/org/opensrp/service/ProductCatalogueServiceTest.java:63]: Yes and that why I dont think the assert makes sense. I was suggesting 
productCatalogue.setPhotoURL("some url");

and the here you assert that the URL is saved

Comment (rehammuzzamil) [reply src/test/java/org/opensrp/service/ProductCatalogueServiceTest.java]: Done

**Review by githengi — APPROVED** [review]

**Review by githengi — APPROVED** [review]

**Pull request merged**